From e1461d67fd2102438630006652c0038915719031 Mon Sep 17 00:00:00 2001
From: Fabio Estevam <fabio.estevam@freescale.com>
Date: Fri, 16 Dec 2011 10:01:05 -0200
Subject: [PATCH 0759/2508] ENGR00170482 ARM: mx6qsabrelite: Fix the total RAM
 size

On mx6qsabrelite there is a total of 1GB of RAM.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabrelite.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabrelite.c b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
index 8295e73..b8b321f 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabrelite.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabrelite.c
@@ -1045,7 +1045,7 @@ static void __init mx6q_sabrelite_reserve(void)
 
 	if (imx6q_gpu_pdata.reserved_mem_size) {
 		phys = memblock_alloc_base(imx6q_gpu_pdata.reserved_mem_size,
-					   SZ_4K, SZ_2G);
+					   SZ_4K, SZ_1G);
 		memblock_free(phys, imx6q_gpu_pdata.reserved_mem_size);
 		memblock_remove(phys, imx6q_gpu_pdata.reserved_mem_size);
 		imx6q_gpu_pdata.reserved_mem_base = phys;
-- 
1.7.7.4

